hollow/src/main/java/com/netflix/hollow/api/producer/HollowProducer.java

```diff
@@ -561,7 +625,7 @@ public void cleanup() {
     }
 
     public static interface Announcer {
-        public void announce(long stateVersion);
+        public void announce(long stateVersion, Map<String, String> metadata);
```

Comment:
Adding metadata tags to announcement feels like unnecessary cognitive load -- we should avoid this. The concept of announcement should remain simple: http://hollow.how/producer-consumer/#announcing-the-state. Specific implementations are free to introduce additional complexity as necessary, but that complexity shouldn't leak into the core producer API.